pontoon/base/models.py

```diff
@@ -1116,8 +1116,8 @@ def has_suggestions(self, locale):
     def unchanged(self, locale):
         return self.with_status_counts(locale).filter(unchanged_count=F('expected_count'))
 
-    def authored_by(self, locale, email):
-        return self.filter(translation__locale=locale, translation__user__email=email)
+    def authored_by(self, locale, id):
+        return self.filter(translation__locale=locale, translation__user__id=id)
```

Comment:
I actually think we should keep the email address in the URL until we fix 1243115, which will allow for filters like `?status=translated&user=usernameORemail`. Using ID (or username) makes it impossible to filter by manually typing the URL.